src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -36,9 +36,7 @@
 import java.net.PasswordAuthentication;
 import java.net.URL;
 import java.net.URLConnection;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Random;
+import java.util.*;
```

Comment:
nit: no * imports
Probably intellij thing. 

Run a google formatter to match style.